src/responder/pam/pamsrv_cmd.c

```diff
@@ -377,6 +377,11 @@ static int pam_parse_in_data_v2(struct pam_data *pd,
                                            body, blen, &c);
                     if (ret != EOK) return ret;
                     break;
+                case SSS_PAM_ITEM_JSON_AUTH_SELECTED:
```

Comment:
Hi,

I think `SSS_PAM_ITEM_JSON_AUTH_SELECTED` and the current `SSS_PAM_ITEM_AUTHTOK` should be mutually exclusive and if both are present it should be a fatal error.

bye,
Sumit

Reply:
Updated